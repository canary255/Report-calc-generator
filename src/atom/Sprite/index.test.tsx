import { describe, expect, test } from "@jest/globals";
import { Sprite } from ".";

describe("Testing RoundedImage component", () => {
  test("RoundedImage component should be defined", () => {
    expect(Sprite).toBeDefined();
  });

  test("RoundedImage component should render correctly", () => {
    expect(Sprite).toMatchSnapshot();
  });
});
