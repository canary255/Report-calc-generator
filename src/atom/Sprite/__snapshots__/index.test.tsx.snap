// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing RoundedImage component RoundedImage component should render correctly 1`] = `[Function]`;
